starlette/applications.py

```diff
@@ -129,7 +131,7 @@ def add_middleware(self, middleware_class: type, **options: typing.Any) -> None:
     def add_exception_handler(
         self,
         exc_class_or_status_code: typing.Union[int, typing.Type[Exception]],
-        handler: typing.Callable,
+        handler: typing.Callable[[Request, Exception], typing.Union[Response, typing.Coroutine[typing.Any, typing.Any, Response]]],
```

Comment:
Should we perhaps use `Awaitable` instead? It looks better. :kissing: 

```suggestion
        handler: typing.Callable[[Request, Exception], typing.Union[Response, Awaitable[Response]]],
```